timed/projects/views.py

```diff
@@ -78,7 +86,15 @@ class CostCenterViewSet(ReadOnlyModelViewSet):
     ]
 
     def get_queryset(self):
-        return models.CostCenter.objects.all()
+        """Don't show cost centers to customers."""
+        user = self.request.user
+
+        queryset = models.CostCenter.objects.all()
+
+        if models.CustomerAssignee.objects.filter(user=user, is_customer=True).exists():
```

Comment:
Deny access by default, only allow if explicitly needed. This code only forbids to customers, but everyone else sees all the data